lib/Dialect/PolyExt/IR/PolyExtDialect.cpp

```diff
@@ -0,0 +1,63 @@
+#include "include/Dialect/PolyExt/IR/PolyExtDialect.h"
+
+#include "include/Dialect/Poly/IR/PolyTypes.h"
+#include "include/Dialect/Poly/IR/Polynomial.h"
+#include "include/Dialect/PolyExt/IR/PolyExtOps.h"
+#include "llvm/include/llvm/ADT/TypeSwitch.h"            // from @llvm-project
+#include "mlir/include/mlir/IR/Builders.h"               // from @llvm-project
+#include "mlir/include/mlir/IR/DialectImplementation.h"  // from @llvm-project
+
+// Generated definitions.
+#include "include/Dialect/PolyExt/IR/PolyExtDialect.cpp.inc"
+#define GET_OP_CLASSES
+#include "include/Dialect/PolyExt/IR/PolyExtOps.cpp.inc"
+
+namespace mlir::heir::poly_ext {
+
+//===----------------------------------------------------------------------===//
+// EncryptedArith dialect.
+//===----------------------------------------------------------------------===//
+
+// Dialect construction: there is one instance per context and it registers its
+// operations, types, and interfaces here.
+void PolyExtDialect::initialize() {
+  addOperations<
+#define GET_OP_LIST
+#include "include/Dialect/PolyExt/IR/PolyExtOps.cpp.inc"
+      >();
+}
+
+namespace {
```

Comment:
nit: I would think this inner namespace is not necessary, since this cpp class won't be part of any other compilation unit.